hvac_service/hvac_gui.py

```diff
@@ -0,0 +1,76 @@
+import dearpygui.dearpygui as dpg
+import dearpygui.demo as demo
+import testclient
+
+def main():
+    client = testclient.HVACTestClient("127.0.0.1:50052")
```

Comment:
```suggestion
    hvac_addr = os.getenv("HVAC_ADDR", "127.0.0.1:50052")
    client = testclient.HVACTestClient(hvac_addr )
```
Let's make this configurable like in testclient.py.